docs/source/modules/vert_coordinate.rst

```diff
@@ -0,0 +1,90 @@
+
+Vertical Coordinates
+====================
+
+Summary
+-------
+
+Climate models split the atmosphere into a number of different levels in order to calculate various quantities (e.g. temperature :math:`T`, zonal wind :math:`u`) throughout the depth of the model. There are a number of ways one may wish to divide the atmosphere; this is the purpose of the ``vert_coordinate`` module. Isca provides four different methods for prescribing model levels:
+
+1. Manually prescribed levels 
+2. Even Sigma levels 
+3. Uneven Sigma levels
+4. Hybrid Scheme
+
+which we will elaborate on later, but first a note about model levels in climate models.
+
+Pressure or Sigma Levels??
```

Comment:
I wonder if you'll need to mention here that the vertical coordinate module actually prescribes the values of the _half_ levels and then subsequently calculates the full levels. So normally there's a value with b=1 at the surface, which is a half level. Then there's some way the model calculates the full levels from the half levels. I'm assuming these are the different solid and dashed lines in your figure?

Reply:
Yes the solid and dashed lines represent the half/full levels.

Good point though, probably best to introduce the concept before detailing the different options.